onnx/backend/test/case/node/crossentropy.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def softmax_2d(x):  # type: (np.ndarray) -> np.ndarray
```

Comment:
Could we make this a full reference implementation
```
def softmax_cross_entropy(x, y, w=None):
  # Softmax
  ...
```
? It'd be even better if you have a Pytorch script showing that this reference implementation matches whatever Pytorch is doing. Please also confirm that Tensorflow can also use this operator.